truffle/src/main/java/org/jruby/truffle/nodes/RubyNode.java

```diff
@@ -211,6 +224,10 @@ public MemoryManager getMemoryManager() {
 
     // ruby() helper
 
+    protected Object ruby(String expression, Object... arguments) {
+        return getContext().inlineRubyHelper(this, expression, arguments);
+    }
```

Comment:
Minor: I would get the frame there and make this one @TruffleBoundary, rather than adding another version in RubyContext.